x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/views/schema.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiPageHeader, EuiButton, EuiPageContentBody } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FlashMessages } from '../../../../shared/flash_messages';
+
+export const Schema: React.FC = () => {
+  return (
+    <>
+      <EuiPageHeader
+        pageTitle={i18n.translate('xpack.enterpriseSearch.appSearch.engine.schema.pageTitle', {
+          defaultMessage: 'Manage engine schema',
+        })}
+        description={i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.schema.pageDescription',
+          { defaultMessage: 'Add new fields or change the types of existing ones.' }
```

Comment:
Related to the ongoing discussion I'm having with Scotty in Slack - we have some overlap here in copy with the Workplace Search team and their schema page. This string in particular is one they use as well, but without a period at the end.

I've currently chosen _not_ to DRY out shared strings and components generally to give us more flexibility with things like this (grammar, casing, styling, what have you). That being said if you feel strongly otherwise, let me know!